client/k8s/k8s_test.go

```diff
@@ -1,263 +0,0 @@
-package k8s
-
-import (
-	"fmt"
-	"testing"
-
-	"github.com/stretchr/testify/assert"
-	"k8s.io/client-go/rest"
-)
-
-func TestGetRawClientConfig(t *testing.T) {
```

Comment:
Not sure if it is useful but we might want to keep some of the tests in the operatorkit lib.